### PR TITLE
Add a Gitter chat badge to README.md

### PR DESCRIPTION
**Pull request by gitter-badger**
### geopandas/geopandas now has a Chat Room on Gitter

@jorisvandenbossche has just created a chat room. You can visit it here: [https://gitter.im/geopandas/geopandas](https://gitter.im/geopandas/geopandas?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&content=body_link).

This pull-request adds this badge to your README.md:

[Gitter](https://gitter.im/geopandas/geopandas?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge)

If my aim is a little off, please [let me know](https://github.com/gitterHQ/readme-badger/issues).

Happy chatting.

PS: [Click here](https://gitter.im/settings/badger/opt-out) if you would prefer not to receive automatic pull-requests from Gitter in future.

### CODE REVIEW

**Comment by kjordahl** [comment]
Hmm, I guess I'd put the badge on the top line with the others, @gitter-badger.

**Comment by kuanb** [comment]
I just noticed this PR. I think this should really be added to the README as soon as possible - the Gitter page is/could be a great resource

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 GeoPandas [![build status](https://secure.travis-ci.org/geopandas/geopandas.png?branch=master)](https://travis-ci.org/geopandas/geopandas) [![Coverage Status](https://coveralls.io/repos/geopandas/geopandas/badge.png)](https://coveralls.io/r/geopandas/geopandas)
 =========
 
+[![Join the chat at https://gitter.im/geopandas/geopandas](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/geopandas/geopandas?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (kuanb) [inline README.md]: If someone with write access could just move this line and append it to line 1, this would be good to go...

**Comment by jorisvandenbossche** [comment]
Done in https://github.com/geopandas/geopandas/commit/5d4a997301255b472b2b67ffdd0c9d422aa3efa1

**Pull request closed without merging**